pkg/src/src/main/scala/edu/berkeley/cs/amplab/sparkr/SparkRSQL.scala

```diff
@@ -0,0 +1,12 @@
+package edu.berkeley.cs.amplab.sparkr
+
+import org.apache.spark.SparkContext
+import org.apache.spark.sql.{SQLContext}
+import org.apache.spark.sql.types.{StructType}
+
+object sparkRSQL {
```

Comment:
All the SparkR scala code follows the Spark code style guide described at https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide -- Relevant to this file is the 2 space indentation.